src/_nebari/stages/infrastructure/__init__.py

```diff
@@ -57,19 +57,14 @@ class DigitalOceanInputVars(schema.Base):
     kubeconfig_filename: str = get_kubeconfig_filename()
 
 
-class GCPGuestAccelerators(schema.Base):
-    name: str
-    count: int
-
-
 class GCPNodeGroupInputVars(schema.Base):
     name: str
     instance_type: str
     min_size: int
     max_size: int
     labels: Dict[str, str]
     preemptible: bool
-    guest_accelerators: List[GCPGuestAccelerators]
+    guest_accelerators: List["GCPGuestAccelerator"]
```

Comment:
why the quotes?

Reply:
It's a forward reference since GCPGuestAccelerator is defined farther down in the file.  Reference: https://peps.python.org/pep-0484/#forward-references

Reply:
Just visually - I think its missing an "s" at the end?

Reply:
> Just visually - I think its missing an "s" at the end?

It's as I intended (no trailing "s").  Check out https://github.com/nebari-dev/nebari/issues/2425 for more context.

Reply:
I see. Thanks for the ref link, @Adam-D-Lewis. I assumed it was something like this, but I was not sure.